packages/template-blank-vue-ts/README.md

```diff
@@ -0,0 +1,23 @@
+# NativeScript-Vue Application
+
+> A native application built with NativeScript-Vue
+
+## Usage
+
+``` bash
+# Install dependencies
+npm install
+
+# Preview on device
+tns preview
+
+# Build, watch for changes and run the application
+tns run
+
+# Build, watch for changes and debug the application
+tns debug <platform>
+
+# Build for production
+tns build <platform> --env.production
+
+```
```

Comment:
Let's base the README off of https://github.com/NativeScript/nativescript-app-templates/blob/master/packages/template-blank-vue/README.md